src/Api/Label/StaticLabelApi.php

```diff
@@ -13,28 +13,30 @@
  */
 class StaticLabelApi extends NullLabelApi
 {
+    private const array LABELS = [
+        'Asset', 'AssetMapper', 'BrowserKit', 'Cache', 'Config', 'Console',
+        'Contracts', 'CssSelector', 'Debug', 'DebugBundle', 'DependencyInjection',
+        'Doctrine', 'DoctrineBridge', 'DomCrawler', 'Dotenv', 'Emoji',
+        'Enhancement', 'ErrorHandler', 'EventDispatcher', 'ExpressionLanguage',
+        'Feature', 'Filesystem', 'Finder', 'Form', 'FrameworkBundle',
+        'HttpClient', 'HttpFoundation', 'HttpKernel', 'Inflector', 'Intl', 'JsonPath', 'JsonStreamer', 'Ldap',
+        'Locale', 'Lock', 'Mailer', 'Messenger', 'Mime', 'MonologBridge', 'Notifier', 'ObjectMapper',
+        'OptionsResolver', 'PasswordHasher', 'PhpUnitBridge', 'Process', 'PropertyAccess',
+        'PropertyInfo', 'ProxyManagerBridge', 'PsrHttpMessageBridge', 'RemoteEvent', 'Routing',
+        'Scheduler', 'Security', 'SecurityBundle', 'Serializer', 'Stopwatch', 'String',
+        'Templating', 'Translation', 'TwigBridge', 'TwigBundle', 'TypeInfo', 'Uid', 'Validator', 'VarDumper',
+        'VarExporter', 'Webhook', 'WebLink', 'WebProfilerBundle', 'WebServerBundle', 'Workflow',
+        'Yaml',
+    ];
+
     public function getComponentLabelsForRepository(Repository $repository): array
```

Comment:
Method needed?

Reply:
I see no call indeed, but maybe let's check this in another PR ?